requirements.txt

```diff
@@ -47,21 +51,21 @@ pycparser==2.14
 pycrypto==2.6.1
 pyOpenSSL==0.15.1
 pyrsistent==0.11.9
-python-cinderclient==1.1.1
+python-cinderclient==1.6.0
```

Comment:
Thanks @stmcginnis and @moshez 

I should have been clearer. 

I meant these are the minimum set of package upgrades in order for `pip` to satisfy all  dependencies after I'd updated to the latest ``python-cinderclient==1.6.1``.

I wasn't particularly scientific. I just upgraded, to latest, every package that was causing a conflict and repeated until I could run ``pip install --editable .[dev]`` cleanly.